app/src/main/res/values/strings.xml

```diff
@@ -118,7 +118,7 @@
   <string name="add_profile_input_confirm_pin">Confirm 3-Digit PIN</string>
   <string name="add_profile_allow_download_heading">Allow Download Access</string>
   <string name="add_profile_allow_download_sub">User is able to download and delete content without Administrator password. Only available with PIN.</string>
-  <string name="add_profile_create_btn">CREATE</string>
+  <string name="add_profile_create_btn">create</string>
```

Comment:
'Create' is fine, too. The all caps fixes this, but if we ever move off of all caps (e.g. migrate to GM2-esque style) then we'd probably want 'Create'.

Reply:
Updated.